HomeKitBridge/accessories/contactSensor.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+class HAPAccessoryContactSensor extends HAPAccessoryBase
+{
+    public function __construct($data)
+    {
+        parent::__construct(
+            $data,
+            [
+                new HAPServiceAccessoryInformation(),
+                new HAPServiceContactSensor()
+            ]
+        );
+    }
+
+    public function readCharacteristicContactSensorState()
+    {
+        return GetValue($this->data['VariableID']);
```

Comment:
Hier bitte das Boolean auf die zwei Zustände von Apple abbilden. Außerdem haben wir in IP-Symcon ja das ".Reversed" Profil... damit kann man denn Zustand umdrehen. Kannst du alles hier abgucken: https://github.com/paresy/HomeKit/blob/master/HomeKitBridge/accessories/smokeSensor.php#L25

Reply:
Kleiner Unterscheid. Der Zustand ist invertiert... Bei uns sind offene Fenster immer True